### PR TITLE
EDU-2225-templates-and-integrations-content

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue: 

[EDU-2225]

### Changes

Add new content related to **Templates and Integrations** section, including:

- Overview
- Undestand templates
- Undertand integrations
- Install an integration
- Update an integration

Use a template guide was already created with Get Started section.

### Additional links

N/A

[EDU-2225]: https://aziontech.atlassian.net/browse/EDU-2225?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/marketplace/templates-and-integrations/overview.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+---
+
+## Implementation
+
+| Scope | Description |
+|---|---|
+| [Azion Marketplace](/en/documentation/products/marketplace/) | Get to know Azion's digital catalog of solutions, including templates and integrations. |
+| [Azion Marketplace first steps](/en/documentation/products/marketplace/first-steps/) | Guide on how to access Azion Marketplace. |
+| [Understand Azion Templates](/en/documentation/products/marketplace/templates/) | Reference to deepen the basics of Azion Templates, how they work, and how you can start using them. |
+| [Understand Azion Integrations](/en/documentation/products/marketplace/integrations/) | Reference to deepen the basics of Azion Integrations, how they work, and how you can start using them. |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/overview.mdx:31]:
```suggestion
| Azion Templates](/en/documentation/products/marketplace/templates/) | Reference to deepen the basics of Azion Templates, how they work, and how you can start using them. |
| Azion Integrations](/en/documentation/products/marketplace/integrations/) | Reference to deepen the basics of Azion Integrations, how they work, and how you can start using them. |
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+Afterward, you can view all your integrations in your **Edge Functions** list. To review your functions, proceed as follows:
+
+1. Access [Real-Time Manager (RTM)](https://manager.azion.com/)
+2. Open the **Products menu** and select **Edge Functions** in the **Edge Libraries** section.
+
+This wiil open the **Edge Functions** list, showing all your functions, including those related to integrations. Select any given function to check the details.
+
+### Integrations’ update process
+
+The integrations presented within the Marketplace aren’t static. Azion and Azion’s partners can update them when they launch new features. Thus, the recommendation is always to have the most updated integration, following the updated schedule of Azion or external providers.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:171]:
```suggestion
The integrations presented within the Marketplace aren’t static. Azion and Azion’s partners can update them when they launch new features. Thus, the recommendation is always to have the most updated integration.
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-integrations.mdx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+<Button href="/en/documentation/products/guides/#marketplace" text="Go to integrations guides" variant="secondary"> 
+
+</Button>
+
+---
+
+## Integrations’ update process
+
+The integrations presented within the Marketplace aren’t static. Azion and Azion’s partners can update them when they launch new features. Thus, the recommendation is always to have the most updated integration, following the updated schedule of Azion or external providers.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-integrations.mdx:164]:
```suggestion
The integrations presented within the Marketplace aren’t static. Azion and Azion’s partners can update them when they launch new features. Thus, the recommendation is always to have the most updated integration.
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-templates.mdx [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+
+By using these templates, the applications run on the Azion Edge Platform, taking advantage of a distributed network to accelerate delivery and a use safer environment. Being part of the Azion's platform, you can also benefit from using other [Azion products](/en/documentation/) to enhance your applications. 
+
+### Easy management
+
+You can update and customize your application settings via RTM as well as add a custom domain for users to access your edge application through it. Additionally, the GitHub repository includes a GitHub Action that you're able to activate and enable continuous deployment and integration between your repository and Azion's platform. 
+
+### Third-party integration
+
+Some templates are integrated with third-party tools to provide a more comprehensive and modern solution. Using this type of template will require compliance with certain requirements such as creating an account or initial setup. In some cases, this integration could generate cost by usage.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-templates.mdx:79]:
```suggestion
Some templates work with third-party tools to provide a more comprehensive and modern solution. Using this type of template will require compliance with certain requirements such as creating an account or initial setup. In some cases, this integration could generate cost by usage.
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+meta_tags: marketplace, edge computing
+namespace: docs_marketplace_install_integrations
+permalink: /documentation/products/marketplace/install-an-integration/
+---
+
+import Button from '~/components/Button.astro';
+
+**Azion Integrations** allows you to improve, compose, or customize your applications. These integrations are based on edge functions and are executed on the edge. This documentation will guide you to install an integration in an existing edge application.
+
+Considering the Azion Marketplace has solutions that run edge functions in two ways, you'll need to proceed according to the type of integration you want to install:
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:13]:
```suggestion
Considering Azion Marketplace has solutions that run edge functions in two ways, you'll need to proceed according to the type of integration you want to install:
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+
+</Button>
+<br />
+<Button href="/en/documentation/products/guides/#marketplace" text="Go to integrations guides" variant="secondary"> 
+
+</Button>
+
+---
+
+## Edge Application Integrations
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:28]:
```suggestion
## Edge Application integrations
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+---
+
+import Button from '~/components/Button.astro';
+
+**Azion Integrations** allows you to improve, compose, or customize your applications. These integrations are based on edge functions and are executed on the edge. This documentation will guide you to install an integration in an existing edge application.
+
+Considering the Azion Marketplace has solutions that run edge functions in two ways, you'll need to proceed according to the type of integration you want to install:
+
+- [Edge Application Integrations](#edge-application-integrations)
+- [Edge Firewall Integrations](#edge-firewall-integrations)
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:16]:
```suggestion
- [Edge Application integrations](#edge-application-integrations)
- [Edge Firewall integrations](#edge-firewall-integrations)
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+</Button>
+
+---
+
+## Edge Application Integrations
+
+These integrations run tasks and services on the edge, closer to the user, employing an existing edge application.
+
+:::tip
+To build a new edge application, go to the [building](/documentation/products/guides/build/build-an-application/) section for more details on this topic.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:33]:
```suggestion
To build a new edge application, go to the [building](/en/documentation/products/guides/build/build-an-application/) section for more details on this topic.
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+To install and instantiate this type of integration, follow the next steps.
+
+### Getting the integration
+
+Access [Real-Time Manager (RTM)](https://manager.azion.com/) and go to **Marketplace** to select the integration you want to install.
+
+After selecting the integration, click the **Get It Now** button, at the bottom-right corner of the page, and a successful message appears to confirm your solution is installed.
+
+:::tip
+You can search by any given integration browsing through the cards or typing a keyword in the search box.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:45]:
```suggestion
You can search any integration by browsing through the cards or typing a keyword in the search box. 
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 10]
```diff
@@ -0,0 +1,175 @@
...
+1. On the upper-left corner, select **Products menu** > **Edge Application** on the **BUILD** section.
+2. On the listing page of your edge applications, select the one you want to use with the integration.
+3. On the application page, open the **Main Settings** tab and enable the Edge Functions switch.
+4. Click the **Save** button.
+
+### Instanting the integration
+
+To add the function and set the Args, still on the application page:
+
+1. Select the **Functions** tab on the top list.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:61]: futuro: acharmos um termo melhor pra top list

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 11]
```diff
@@ -0,0 +1,175 @@
...
+2. Click the **Add Function** button.
+3. Choose an easy to remember name for your function.
+4. On the dropdown function menu, select the function related to your intagration.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:64]:
```suggestion
4. On the dropdown function menu, select the function related to your integration.
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 12]
```diff
@@ -0,0 +1,175 @@
...
+
+This action will load the function, showing a form with the function’s code and, just above it, two tabs: **Code** and **Args**.
+
+On the Code tab and you’ll be able to navigate through the code, but won’t be able to change it. On the Args tab, you’ll pass the solution parameters.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:68]:
```suggestion
On the **Code** tab you’ll be able to navigate through the code, but won’t be able to change it. On the **Args** tab, you’ll pass the solution parameters.
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 13]
```diff
@@ -0,0 +1,175 @@
...
+
+:::tip
+Go to [Azion's guides page ](/en/documentation/products/guides/#marketplace) to read the specifications for each integration.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:71]:
```suggestion
Go to [Azion's guides page](/en/documentation/products/guides/#marketplace) to read the specifications for each integration.
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 14]
```diff
@@ -0,0 +1,175 @@
...
+To install and instantiate this type of integration, follow the next steps.
+
+### Getting the integration
+
+Access [Real-Time Manager (RTM)](https://manager.azion.com/) and go to **Marketplace** to select the integration you want to install.
+
+After selecting the integration, click the **Get It Now** button, at the bottom-right corner of the page, and a successful message appears to confirm your solution is installed.
+
+:::tip
+You can search by any given integration browsing through the cards or typing a keyword in the search box.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:103]:
```suggestion
You can search any integration by browsing through the cards or typing a keyword in the search box. 
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 15]
```diff
@@ -0,0 +1,175 @@
...
+To instantiate your integration, still on the Edge Firewall page:
+
+1. Select the **Functions** tab.
+2. Click the **Add Function** button.
+3. Give an easy to remember name to your instance.
+4. On the dropdown menu, select the function related to your integration.
+
+This action will load the function, showing a form with the function’s code and, just above it, two tabs: **Code** and **Args**.
+
+On the Code tab and you’ll be able to navigate through the code, but won’t be able to change it. On the Args tab, you’ll pass the solution parameters.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:128]:
```suggestion
On the **Code** tab you’ll be able to navigate through the code, but won’t be able to change it. On the **Args** tab, you’ll pass the solution parameters.
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 16]
```diff
@@ -0,0 +1,175 @@
...
+
+:::tip
+Go to [Azion's guides page ](/en/documentation/products/guides/#marketplace) to read the specifications for each integration.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:131]:
```suggestion
Go to [Azion's guides page](/en/documentation/products/guides/#marketplace) to read the specifications for each integration.
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 17]
```diff
@@ -0,0 +1,175 @@
...
+2. Click the **New Rule** button.
+3. Give a name to the rule.
+4. Select a *criteria* to run and catch the domain you want to run the solution on. Example: `if Hostname is equal xxxxxxxxxxxx.map.azionedge.net`.
+5. Below, select a *behavior* to the criteria. In this case, it’ll be **Run Function**.
+- Select the adequate function according to the name you gave it in the instantiate step.
+6. Click the **Save** button.
+
+Done. Now your integration is running for every request made to the domain you indicated.
+
+### Third-Party integration
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:150]:
```suggestion
### Third-party integration
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 18]
```diff
@@ -0,0 +1,175 @@
...
+
+Some integrations are integrated with third-party tools to provide a more comprehensive and modern solution. Using this type of integration will require compliance with certain requirements such as creating an account or initial setup. In some cases, this integration could generate cost by usage.
+
+:::tip
+Go to [Azion's guides page ](/en/documentation/products/guides/#marketplace) to read the specifications for each integration.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:155]:
```suggestion
Go to [Azion's guides page](/en/documentation/products/guides/#marketplace) to read the specifications for each integration.
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 19]
```diff
@@ -0,0 +1,175 @@
...
+:::
+
+---
+
+## Managing an integration
+
+Afterward, you can view all your integrations in your **Edge Functions** list. To review your functions, proceed as follows:
+
+1. Access [Real-Time Manager (RTM)](https://manager.azion.com/)
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:164]:
```suggestion
1. Access [Real-Time Manager (RTM)](https://manager.azion.com/).
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 20]
```diff
@@ -0,0 +1,175 @@
...
+2. Open the **Products menu** and select **Edge Functions** in the **Edge Libraries** section.
+
+This wiil open the **Edge Functions** list, showing all your functions, including those related to integrations. Select any given function to check the details.
+
+### Integrations’ update process
+
+The integrations presented within the Marketplace aren’t static. Azion and Azion’s partners can update them when they launch new features. Thus, the recommendation is always to have the most updated integration.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:171]:
```suggestion
The integrations presented within Marketplace aren’t static. Azion and Azion’s partners can update them when they launch new features. Thus, the recommendation is to always have the most updated integration.
```

Comment (hannah-sk) [reply src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:171]: updated integration version?

File: src/content/docs/en/pages/marketplace/templates-and-integrations/overview.mdx [hunk 21]
```diff
@@ -0,0 +1,60 @@
...
+meta_tags: marketplace, edge computing
+namespace: docs_marketplace_templates_integrations_overview
+permalink: /documentation/products/marketplace/templates-and-integrations-overview/
+---
+
+import Button from '~/components/Button.astro';
+
+Azion provides a wide range of solutions to speed up your workflow: from development and deployment to management and monitoring. These solutions include [pre-built templates](#azion-templates) to deploy a new project from scratch and [integrations](#azion-integrations) to increase the capabilities of your applications and services.
+
+Through **Azion Edge Platform**, you have access to edge-running software built by Azion's specialized teams and third-party sellers and partners using [Azion Marketplace](https://www.azion.com/en/documentation/products/marketplace/) as a platform to distribute their products.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/overview.mdx:13]:
```suggestion
Through **Azion Edge Platform**, you have access to edge-running software built by Azion's specialized teams and third-party sellers and partners using [Azion Marketplace](/en/documentation/products/marketplace/) as a platform to distribute their products. 
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-integrations.mdx [hunk 22]
```diff
@@ -0,0 +1,167 @@
...
+- **Security**: authenticate users, validate input data, route requests, define rate limiting, and scan and filter content to protect your applications from malicious attacks.
+- **Performance**: reduce the load on your servers, minimize network latency, and accelerate content delivery, providing an improved experience for users.
+- **Database**: manage your databases in a smooth way and generate dynamic content as demanded, such as personalized recommendations or geolocalized messages, based on user preferences or real-time data.
+- **Edge AI**: implement facial recognition and ID matching tools, and accelerate and automate processes with state-of-the-art integrations.
+
+---
+
+## Main configurations and characteristics
+
+### User-friendly setting up
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-integrations.mdx:45]:
```suggestion
### User-friendly set up
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-integrations.mdx [hunk 23]
```diff
@@ -0,0 +1,167 @@
...
+
+You can install and instantiate the integration in a few steps through [Real-Time Manager (RTM)](https://manager.azion.com/), Azion's configuration interface, and quickly improve your applications. 
+
+### Edge capabilities
+
+Based on edge functions, these integrations run on the Azion Edge Platform, taking advantage of a distributed network to accelerate delivery and a use safer environment. Being part of Azion's platform, you can also benefit from using other [Azion products]().
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-integrations.mdx:51]:
```suggestion
Based on edge functions, these integrations run on the Azion Edge Platform, taking advantage of a distributed network to accelerate delivery and a use safer environment. Being part of Azion's platform, you can also benefit from using other [Azion products](/en/documentation/). 
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-integrations.mdx [hunk 24]
```diff
@@ -0,0 +1,167 @@
...
+| Integration | Description |
+|---|---|
+| [Azion Bot Protection](/en/documentation/products/guides/bot-protection/) | Use this integration to analyze incoming requests and assign a score based on rules and behaviors. |
+| [Radware Bot Manager](/en/documentation/products/guides/radware/) | Defend your online assets against bot attacks. |
+
+- **Security optimization and access control**: increase your security and access control measures with threat response, user authentication, and granular control over access to your applications and content, ensuring a robust defense against attacks.
+
+| Integration | Description |
+|---|---|
+| [Send Event to Endpoint](/en/documentation/products/guides/send-event/) | Stream request data to an HTTP endpoint, taking the request data and transmitting it to a user-defined endpoint via Javascript fetch API. |
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-integrations.mdx:139]:
```suggestion
| [Send Event to Endpoint](/en/documentation/products/guides/send-event/) | Stream request data to an HTTP endpoint, taking the request data and transmitting it to a user-defined endpoint via JavaScript fetch API. |
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-integrations.mdx [hunk 25]
```diff
@@ -0,0 +1,167 @@
...
+| [IP Address Reputation](/en/documentation/products/guides/ipaddress/) | This integration uses a reputation score for a given IP address, provided by IPQualityScore. |
+
+- **Workloads and payloads**: manage incoming data sizes efficiently and prevent resource overload using these integrations.
+
+| Integration | Description |
+|---|---|
+| [Limit Payload Size](/en/documentation/products/guides/limitpayload/) | Through this integration, evaluate request data and deny payloads that exceed a predefined limit. |
+| [Massive redirect](/en/documentation/products/guides/how-to-massive-redirection/) | Handle a massive quantity of domain redirects, such as domain migrations. |
+| [Process Request Data Into Headers](/en/documentation/products/guides/requestdata/) | Stop a request whenever a request body field is empty. |
+| [Request Variation Controller](/en/documentation/products/guides/requestvariation/) | Create a signed cookie to track the different arguments  the user (or the origin) is using across the requests and check the number of variations made to block access to the origin whenever the user exceeds the maximum number permitted. |
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-integrations.mdx:154]:
```suggestion
| [Request Variation Controller](/en/documentation/products/guides/requestvariation/) | Create a signed cookie to track the different arguments the user (or the origin) is using across the requests and check the number of variations made to block access to the origin whenever the user exceeds the maximum number permitted. |
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-templates.mdx [hunk 26]
```diff
@@ -0,0 +1,243 @@
...
+
+:::caution[warning]
+To guarantee the optimal performance of certain templates, some Azion products must be activated in your account and you must enable them separately via RTM. If these products are activated, the execution of the templates could generate usage-related costs. Check the [guides by template](/en/documentation/products/guides/#azion-templates) and the [pricing page](https://www.azion.com/en/pricing/) for more information.
+:::
+
+---
+
+## Main configurations and characteristics
+
+### User-friendly setting up
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-templates.mdx:55]:
```suggestion
### User-friendly set up
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-templates.mdx [hunk 27]
```diff
@@ -0,0 +1,243 @@
...
+
+After selecting a template that matches your needs, you must only complete a few steps to quickly deploy the template to the edge through RTM, Azion's configuration interface. With this, you don't need to start from scratch and can have a project running in a few minutes.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-templates.mdx:57]:
```suggestion
After selecting a template that matches your needs, you only have to complete a few steps to quickly deploy the template to the edge through RTM, Azion's configuration interface. With this, you don't need to start from scratch and can have a project running in a few minutes.
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-templates.mdx [hunk 28]
```diff
@@ -0,0 +1,243 @@
...
+
+</Button>
+<br />
+<Button href="#" text="deploy this template" variant="secondary"> 
+
+</Button>
+
+#### React Boilerplate
+
+The Azion React Boilerplate helps you build a React Single-Page Application (SPA) directly on the edge By executing the boilerplate, several steps are automated, from repository management to edge deployment.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-templates.mdx:133]:
```suggestion
The Azion React Boilerplate helps you build a React Single-Page Application (SPA) directly on the edge By executing the boilerplate. Several steps are automated, from repository management to edge deployment.
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-templates.mdx [hunk 29]
```diff
@@ -0,0 +1,243 @@
...
+
+</Button>
+
+### Microservices and API usage
+
+Employ rules and capabilities that help to control and manage incoming traffic and, this way, ensure a more efficient resource allocation and safer environment for your applications.
+
+#### Upstash Rate Limiting
+
+The Upstash Rate Limiting template helps you to implement rate limiting in a serverless environment, without requiring extensive manual setup. This template includes configurations for limiting requests and window limit and works in conjunction with Azion Edge Functions,
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-templates.mdx:235]:
```suggestion
The Upstash Rate Limiting template helps you to implement rate limiting in a serverless environment, without requiring extensive manual setup. This template includes configurations for limiting requests and window limit and works in conjunction with Azion Edge Functions.
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/update-an-integration.mdx [hunk 30]
```diff
@@ -0,0 +1,50 @@
...
+title: How to update an integration
+description: This documentation will guide you to check the availability of an update and how to install it.
+meta_tags: marketplace, edge computing
+namespace: docs_marketplace_update_integrations
+permalink: /documentation/products/marketplace/update-an-integration/
+---
+
+import Button from '~/components/Button.astro';
+
+The integrations included within the Marketplace aren’t static. Azion and Azion’s partners can update them when they launch new features. Thus, the recommendation is always to have the most updated integration.
```
[the remark this answers is not in the document]
Comment (hannah-sk) [reply src/content/docs/en/pages/marketplace/templates-and-integrations/update-an-integration.mdx:11]: version?

File: src/content/docs/en/pages/marketplace/templates-and-integrations/update-an-integration.mdx [hunk 31]
```diff
@@ -0,0 +1,50 @@
...
+</Button>
+
+---
+
+## Checking the availability of an update
+
+To make sure that you’re running the latest version of an integration:
+
+1. Access [Real-Time Manager (RTM)](https://manager.azion.com/) and go to **Marketplace**.
+2. On the Marketplace's homepage, select the solution you want to check.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/update-an-integration.mdx:26]:
```suggestion
2. On Marketplace's homepage, select the solution you want to check.
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx [hunk 32]
```diff
@@ -0,0 +1,175 @@
...
+1. On the upper-left corner, select **Products menu** > **Edge Application** on the **BUILD** section.
+2. On the listing page of your edge applications, select the one you want to use with the integration.
+3. On the application page, open the **Main Settings** tab and enable the Edge Functions switch.
+4. Click the **Save** button.
+
+### Instanting the integration
+
+To add the function and set the Args, still on the application page:
+
+1. Select the **Functions** tab on the top list.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/install-an-integration.mdx:61]:
```suggestion
1. Select the **Functions** tab.
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-templates.mdx [hunk 33]
```diff
@@ -0,0 +1,243 @@
...
+The Azion Next.js Static Boilerplate enables the creation and deployment of a Next.js Single-Page Application (SPA) directly on the edge of the network.
+
+<Button href="/en/documentation/products/guides/nextjs-static-boilerplate/" text="go to the Next.js Static Boilerplate guide" variant="secondary"> 
+
+</Button>
+<br />
+<Button href="#" text="deploy this template" variant="secondary"> 
+
+</Button>
+
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-templates.mdx:130]:
```suggestion
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-integrations.mdx [hunk 34]
```diff
@@ -0,0 +1,169 @@
...
+
+## Main configurations and characteristics
+
+### User-friendly set up
+
+You can install and instantiate the integration in a few steps through [Real-Time Manager (RTM)](https://manager.azion.com/) and quickly improve your applications. 
+
+### Edge capabilities
+
+Based on edge functions, these integrations run on the **Azion Edge Platform**, taking advantage of a distributed network to accelerate delivery and a use safer environment. Being part of Azion's platform, you can also benefit from using other [Azion products](/en/documentation/).
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-integrations.mdx:51]:
```suggestion
Based on edge functions, these integrations run on the **Azion Edge Platform**, taking advantage of a distributed network to accelerate delivery and a safer environment. Being part of Azion's platform, you can also benefit from using other [Azion products](/en/documentation/). 
```

**Review by hannah-sk — APPROVED** [review]

File: src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-integrations.mdx [hunk 35]
```diff
@@ -0,0 +1,169 @@
...
+
+---
+
+## Implementation
+
+| Scope | Description |
+|---|---|
+| [Azion Marketplace](/en/documentation/products/marketplace/) | Get to know Azion's digital catalog. |
+| [Install an integration](/en/documentation/products/start-with-a-template/) | Step-by-step on how to use and install Azion Integrations. |
+| [Azion Templates guides](/en/documentation/products/guides/#marketplace) | Find the dedicated guide for each integration. |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/understand-azion-integrations.mdx:23]:
```suggestion
| [Azion Integrations guides](/en/documentation/products/guides/#marketplace) | Find the dedicated guide for each integration. |
```

File: src/content/docs/en/pages/marketplace/templates-and-integrations/update-an-integration.mdx [hunk 36]
```diff
@@ -0,0 +1,50 @@
...
+
+---
+
+## About your control over the version
+
+When you update a integration in Azion Marketplace, *it won’t overwrite the version of the integration you’re currently running on your edge application*. This occurs to give you control over the version you want to use with your application.
+
+For instance, you could test a new version of an integration with a different edge application. So, when the new version has passed through your validation process, on your edge application, you can select it to run. To do so, change the integration version on the functions dropdown menu of your edge application.
+
+<Button href="/en/documentation/products/guides/build/configure-main-settings/" text="go to configuring main settings guide" variant="secondary">
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/marketplace/templates-and-integrations/update-an-integration.mdx:48]:
```suggestion
<Button href="/en/documentation/products/guides/build/instantiate-edge-functions/" text="go to How to instantiate edge functions guide" variant="secondary"> 
```

**Review by bru-andrade — APPROVED** [review]

**Pull request merged**